src/accuraterip-checksum.c

```diff
@@ -2,11 +2,11 @@
  ============================================================================
  Name        : accuraterip-checksum.c
  Author      : Leo Bogert (http://leo.bogert.de)
- Git         : http://leo.bogert.de/accuraterip-checksum
- Version     : See global variable "version"
- Copyright   : GPL
- Description : A C99 commandline program to compute the AccurateRip checksum of singletrack WAV files.
- 	 	 	   Implemented according to http://www.hydrogenaudio.org/forums/index.php?showtopic=97603
+ Git         : https://github.com/leo-bogert/accuraterip-checksum
```

Comment:
This is no longer accurate: the code in this `accuraterip-checksum.c` is not the same in the linked repository.